bin/chandra_repro

```diff
@@ -137,6 +137,63 @@ def error_out(params,msg,val):
     print("\n%s" % msg)
     raise ValueError("%s" % val)
 
+
+def filter_aspsolobi_from_aspsol(asol_list,mydir):
+    """As of DS10.8.3, there are now two flavors of aspect solution 
+    files, per OBI (CONTENT=ASPSOLOBI) and the original per-constant
+    aspect interval (cai) (CONTENT=ASPSOL).  Users should only ever
+    have one or the other. 
+    
+    Unfortunately, due to a quirk in the archive, users may have both.
+    
+    We know that there can be 1 and only 1 ASPSOLOBI file per obi (multi-obi
+    obsid's will need to have already been split, #repro5). 
+    
+    So if we find both ASPSOLOBI and ASPSOL files, we are going to pick
+    the ASPSOLOBI files.  This provides a better off-axis angle value
+    and tighter FOV.
+    """
+    from pycrates import read_file
+
+    content_values = {}
+
+    for f in asol_list:
+        tab = read_file(f.strip("\n").strip(" "), mode="r")
+        content = tab.get_key_value("CONTENT")
+        if content in content_values:
+            content_values[content].append(f)
+        else:
+            content_values[content] = [f]
+        
+    if "ASPSOL" in content_values and "ASPSOLOBI" in content_values:
+        v0("WARNING: The primary directory has both CONTENT=ASPSOL and "+
+          "CONTENT=ASPSOLOBI files. Only the ASPSOLOBI file "+
+          "{} will be used".format(content_values["ASPSOLOBI"]))
+        # don't return yet, fall through to next check
+
+        # But I do want to go ahead and remove the old files from the
+        # output directory.
+        for oldasp in content_values["ASPSOL"]:
+            old = os.path.join(mydir, 
+                      os.path.basename(oldasp.strip("\n").strip(" ")))
+            if os.path.exists(old):
+                os.unlink(old)
+
+    if "ASPSOLOBI" in content_values:
+        if len( content_values["ASPSOLOBI"] ) > 1:
```

Comment:
Here's some "proper" questions (that really relate to different parts of the code than here...)

a) I think this should this check be moved to before the check on line 168 - i.e. immediately error out

b) I'm wondering if it's worth warning message on line 169 - how about if we just always use ASPSOLOBI in preference and don't bother telling the user in this case (it would mean we don't have to explain what the message means to people, and if a user really wants to be using the ASPSOL rather than ASPSOLOBI file I think it's okay for us to say they will have to do things manually / have the directory set up correctly [as we don't expect this to be a common occurrence]).

Reply:
Okay, I've moved the `>1` check earlier.

I've kept the warning but up'ed it to `verbose=2`.